jujugui/static/gui/src/app/components/spinner/_spinner.scss

```diff
@@ -1,21 +1,72 @@
 @keyframes spinner {
   0% {
-    @include rotate(0deg);
+    transform: rotate(0deg);
   }
   100% {
-    @include rotate(360deg);
+    transform: rotate(360deg);
```

Comment:
Yep, we could probably remove these mixins as our supported browsers don't need the vendor prefixes.